server/reports/attendance/attendance-endpoint.js

```diff
@@ -73,64 +67,62 @@ export function attendanceRequestHandler(request, response) {
   MindbodyAccess.getAuth()
     .then((value) => {
       MindbodyAccess.authToken = value.AccessToken;
-      // gets all classes between StartDateTime - EndDateTime
-      return MindbodyAccess.getClasses({
-        StartDateTime: startdate,
-        EndDateTime: enddate,
-      });
+      return MindbodyAccess.getClasses();
```

Comment:
Any reason the startdate and enddate parameters were removed?
With the way the getClasses API call works, when those parameters aren't specified, it sets them both to the current date by default.